CHANGELOG.md

```diff
@@ -20,6 +20,9 @@
 - \#2663 - Adapt filter count, reflect current result set
 - \#2702 - Show proper info on app creation auth error
 
+### Fixed
+- \#2684 - Running instances are confusing
```

Comment:
It's somehow considered as a bug, thus I would keep it in the Fixed section. What do you think?